wallet/db.c

```diff
@@ -1697,7 +1697,7 @@ void db_bind_preimage(struct db_stmt *stmt, int pos, const struct preimage *p)
 
 void db_bind_sha256(struct db_stmt *stmt, int pos, const struct sha256 *s)
 {
-	db_bind_blob(stmt, pos, s->u.u8, sizeof(struct sha256));
+        db_bind_blob(stmt, pos, s->u.u8, sizeof(struct sha256));
```

Comment:
This is a no-op change, could you remove it just for cleanliness' sake? :sunglasses:

Reply:
ups! I think my emacs has some problem  with the space indentation (?) I didn't see it before on my editor.

Anyway, it is a good catch, space is important. Thanks!